### PR TITLE
New launchpad design

### PR DESCRIPTION
**Pull request by nmboavida**

### CODE REVIEW

File: sources/launchpad/market/fixed_price.move [hunk 1]
```diff
@@ -0,0 +1,254 @@
+/// Module of a Fixed Initial NFT Offering `Config` type.
+/// 
+/// It implments a fixed price launchpad configuration.
+/// 
+/// TODO: When deleting the slingshot, we should guarantee that no nft is still
```
Comment (bausano) [inline sources/launchpad/market/fixed_price.move:5]: Do we want to delete it?

File: sources/launchpad/market/fixed_price.move [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    use nft_protocol::sale::{Self, NftCertificate};
+
+    struct FixedPriceMarket has drop {}
+    
+    struct Market has key, store {
+        id: UID,
+        price: u64,
+    }
+
+    public entry fun create_single_market(
```
Comment (bausano) [inline sources/launchpad/market/fixed_price.move:29]: Not immediately clear how `single_market` differs to `multi_market`

File: sources/launchpad/market/fixed_price.move [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    //     } = config;
+
+    //     object::delete(id);
+    // }
+
+    // // === Modifier Functions ===
+
+    // /// Permissioned endpoint to be called by `admin` to edit the fixed price 
+    // /// of the launchpad configuration.
+    // public entry fun new_price<T, Config>(
```
Comment (bausano) [inline sources/launchpad/market/fixed_price.move:220]: TODO

Comment (nmboavida) [reply sources/launchpad/market/fixed_price.move]: Added

**Review by bausano — APPROVED** [review]
Launchpad still needs some work but the current design in the right direction

**Pull request merged**